dissect/regf/regf.py

```diff
@@ -478,7 +478,12 @@ def try_decode_sz(data):
 
         # This will return the string utf-16-le decoded up until the first
         # double NULL byte.
-        return data.split(b"\x00\x00")[0].decode("utf-16-le")
+        for idx in range(0, len(data), 2):
```

Comment:
Please add a little comment to clarify why the old split is not sufficient, otherwise it's bound to re-appear in the future at some point I'm afraid.